oauth_provider/security/oauth_provider_security.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        ##############################################################################
```

Comment:
This is part of our own templates since at least v5, I think this is time to update our templates.
I'll change that.